bam_addrprg.c

```diff
@@ -360,9 +360,8 @@ static bool init(const parsed_opts_t* opts, state_t** state_out) {
         // Check does not already exist
         kstring_t hdr_line = { 0, 0, NULL };
         if (sam_hdr_find_line_id(retval->output_header, "RG", "ID", opts->rg_id, &hdr_line) == 0) {
-            fprintf(stderr, "[init] ID of new RG line specified conflicts with that of an existing header RG line. Overwrite not yet implemented.\n");
-            free(hdr_line.s);
-            return false;
+            sam_hdr_remove_line_id(retval->output_header, "RG", "ID", opts->rg_id);
+            fprintf(stderr, "[init] ID of new RG line specified found, we will replace the old ID line with new one.\n");
```

Comment:
Since it is no longer considered an erroneous path, printing to `stderr` should be removed. Maybe add an error message in case `sam_hdr_remove_line_id` fails.

Reply:
I totally agree with that, but I want to print some information to remind user you replace the old RG line with the new one, how should I do to achieve the goal without print to stderr